src/quo2/components/tags/number_tag/view.cljs

```diff
@@ -0,0 +1,25 @@
+(ns quo2.components.tags.number-tag.view
+  (:require [quo2.components.icon :as icons]
+            [quo2.components.markdown.text :as text]
+            [react-native.core :as rn]
+            [quo2.components.tags.number-tag.style :as style]
+            [quo2.theme :as quo.theme]))
+
+(defn number-tag-internal
+  [{:keys [number size-key blur? theme] :as props}]
+  (let [size      (get-in style/sizes [size-key :size])
+        icon-size (get-in style/sizes [size-key :icon-size])]
+    [rn/view (style/container props)
+     (if (and (> size 20) (< (count number) 3))
+       [text/text
+        {:size   (if (= size 32)
+                   :paragraph-2
+                   :label)
+         :weight :medium
+         :style  {:color (style/get-color blur? theme)}}
+        (str "+" number)]
```

Comment:
I wonder should this be an `i18n` label with a param? 🤔

Reply:
guess designers would know that, i.e will it ever be on the other side etc

Reply:
Seems more like math than language, probably wouldn't change IMO